exampleCourse/questions/element/orderBlocks/info.json

```diff
@@ -2,5 +2,6 @@
   "uuid": "d5fa5e91-0230-444e-9d42-53ce9713e36c",
   "title": "Element pl-order-blocks: Dragging blocks to form the solution of a problem",
   "topic": "Element",
-  "type": "v3"
+  "type": "v3",
+  "authors": ["nami5504@gmail.com", "seth.poulsen@usu.edu", "dhsmith2@illinois.edu"]
```

Comment:
I just happened to know the authors of this question. 

Are there other example course questions I should put them on too?

Reply:
I can take all the author info that was listed in tags and move it into authors. (for test & example course).

Reply:
Thinking about it more, this should definitely be a separate PR. 

In this PR, we should add the authors field on a few questions as an example, but trying to edit the 100s of questions in test and example course should go in a separate PR.

Reply:
we should also use David's more up to date email here now; [dhsmith4@vt.edu](mailto:dhsmith4@vt.edu)